certified-connectors/Zuva DocAI/apiDefinition.swagger.json

```diff
@@ -248,9 +248,9 @@
     },
     "/fields": {
       "get": {
-        "summary": "Get All Fields",
+        "summary": "Get Field List",
         "description": "Get a list of all fields available to you.",
-        "operationId": "GetAllFields",
```

Comment:
@ben741 The operation has been deleted. This change breaks existing flows. changing operation names might break existing flows.
Can you please confirm that no users will be affect from this change?

Reply:
The connector is not certified or deployed yet, so no users should be affected.